### ISSUE TITLE
Training on Coco dataset from scratch

### ISSUE THREAD

**Issue by guskarls** [qqwweee/keras-yolo3#703]
Hi,

Has anyone been able to train on the coco data from scratch successfully? When I am training the train loss stops improving at 60 and I get an mAP of 0.1. Also if I use the pretrained weights from the darknet webpage for yolov3, I get a loss around 45. Shouldn't this be close to 0 as these are the weights that have been trained on the coco dataset? With the pretrained weights, I get mAP around 0.55 which is correct. Does this mean something is wrong with the loss function? Feel like it should be close to zero..

Anyone that has experienced the same or know what the loss should be?

**Comment by guskarls** [comment]
To clarify, I am not using pretrained weights when training the model (and no freezing of layers).

**Comment by v5wow** [comment]
Hello, have you solved this problem? I also encountered this problem. I didn't use the pretrained weights, and the final loss is around 20, the mAP is 15%.

**Comment by mordechail** [comment]
> Hi,
> 
> Has anyone been able to train on the coco data from scratch successfully? When I am training the train loss stops improving at 60 and I get an mAP of 0.1. Also if I use the pretrained weights from the darknet webpage for yolov3, I get a loss around 45. Shouldn't this be close to 0 as these are the weights that have been trained on the coco dataset? With the pretrained weights, I get mAP around 0.55 which is correct. Does this mean something is wrong with the loss function? Feel like it should be close to zero..
> 
> Anyone that has experienced the same or know what the loss should be?

Hello,
Can you give more details how exactly you reach 0.55 mAP with pre trained weights? I tried this on some coco validation set and coco weights and I got only 0.13.
Thanks
